napari/layers/image/image.py

```diff
@@ -239,6 +245,10 @@ def __init__(
             rendering=Event,
             iso_threshold=Event,
             attenuation=Event,
+            render_as_plane=Event,
+            plane_position=Event,
+            plane_normal=Event,
+            plane_thickness=Event,
```

Comment:
I don't think we need these events as they are in the `Plane3D` `EventedModel` right?

Reply:
good catch! removed